public/catalogue-frontend.css

```diff
@@ -12376,4 +12376,17 @@ table.shutter-events {
   transition: height .5s;
   overflow: hidden;
   height:120px; /* If you change this, change the corresponding smallHeight value in catalogue-frontend.css too */
+}
+
+.env-maj-version-behind{
```

Comment:
Is it any version behind, not just major versions?

Reply:
Yeah it's any version, good spot